Extending admin UI update (#950)
ezsystems/developer-documentation: docs/tutorials/extending_admin_ui/1_creating_a_dashboard_tab.md

# Step 1 - Creating a Dashboard tab

The Dashboard is the front page that you visit after logging in to the Back Office.

> You can also get to it from any other page by selecting eZ logo in the top left corner.

Reviewer: ```suggestion
You can also get to it from any other page by selecting the site in the top left corner.
```
Safer option, in case a customer has customized the BO to use their own logo

> By default, Dashboard contains two blocks: "My content" and "Common content", which list Content items and Media.

Reviewer: ```suggestion
By default, the Dashboard contains two blocks: "My content" and "Common content", which list Content items and Media.
```

    If you cannot see the results or encounter an error, clear the cache and reload the application.

> At this point you can go to the Dashboard in the Back Office, select eZ logo in the top left corner.

Reviewer: ```suggestion
At this point you can go to the Dashboard in the Back Office, select the site logo in the top left corner.
```
